test/selenium-test/sign-in-widget/login.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, NavigationStart} from '@angular/router';
+
+import { OktaAuthService } from '@okta/okta-angular';
+import * as OktaSignIn from '@okta/okta-signin-widget';
+import { environment } from '../environments/environment';
+
+@Component({
+  selector: 'app-secure',
+  template: `
+    <!-- Container to inject the Sign-In Widget -->
+    <div id="okta-signin-container"></div>
+  `
+})
+export class LoginComponent {
+  authService;
+  widget = new OktaSignIn({
+    el: '#okta-signin-container',
+    baseUrl: `https://${environment.yourOktaDomain}`,
+    authParams: {
+      pkce: true
+    },
+    clientId: environment.clientId,
+    redirectUri: 'http://localhost:9000/login/callback'
```

Comment:
Does the org need to be configured with this port? Can this app be configured to use another port (say, 3000)?